src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] inputArray = readFromFile(fromFileName).split(",");
+        int buy = 0;
+        int supply = 0;
+        int result;
+
+        for (int i = 0; i < inputArray.length; i++) {
+            if (inputArray[i].equals("buy")) {
+                buy += Integer.parseInt(inputArray[i + 1]);
+            } else if (inputArray[i].equals("supply")) {
+                supply += Integer.parseInt(inputArray[i + 1]);
+            }
+        }
+        result = supply - buy;
+        StringBuilder stringBuilder = new StringBuilder().append("supply,").append(supply)
+                .append(System.lineSeparator()).append("buy,").append(buy)
+                .append(System.lineSeparator()).append("result,").append(result);
+        String calculateResult = stringBuilder.toString();
+
```

Comment:
extract this logic to a private method too, so your public method looks tidy and concise